src/v0/destinations/customerio/util.js

```diff
@@ -204,7 +204,9 @@ const groupResponseBuilder = (message) => {
   if (validateEmail(id)) {
     cioProperty = "email";
   }
-  rawPayload.cio_relationships.push({ identifiers: { [cioProperty]: id } });
+  if (isDefinedAndNotNull(id)){
```

Comment:
Introduced this condition for backward compatibility as before we were not mapping any `id` in case no identifier is found

Reply:
Makes sense